src/renderer/component/categoryList/view.jsx

```diff
@@ -1,11 +1,11 @@
 // @flow
-import React from 'react';
+import * as React from 'react';
```

Comment:
why this change?

Reply:
It's needed after adding `React.Fragment`. Flow acts funny without it.